lib/tasks/puppet.rake

```diff
@@ -57,4 +57,32 @@ namespace :puppet do
       end
     end
   end
+
+  namespace :migrate do
+    desc "
+    Migrate Puppet provisioning parameters from ENC to Host parameters.
+    This task is relevant if you want to continue using Puppet provisioning,
+    but do not want to install the Puppet plugin and get the full ENC funcionality."
+
+    task :to_parameters => :environment do
```

Comment:
I wonder if this isn't too generic name for the task, perhaps `migrate:puppet_config_to_parameters` ?

Reply:
or perhaps have this task under the `puppet` namespace?

Reply:
it is in the `puppet` namespace `puppet:migrate:to_parameters`

Reply:
Ah ok, I haven't expanded the context enough, ignore me.